v1/website/versioned_docs/version-1.0.15/guides-custom-pages.md

```diff
@@ -34,12 +34,108 @@ root-of-repo
 │   └── static
 ```
 
+
```

Comment:
Can you add this documentation to https://github.com/facebook/Docusaurus/blob/master/docs/api-doc-markdown.md instead?